### PR TITLE
RAC-6100 Add graph for ucs poller and its unit test

### PR DESCRIPTION
**Pull request by mcgG**
RAC-6100 Add graph for ucs poller and its unit test
----

Add ucs poller to discovery graph

jenkins depends on:
https://github.com/RackHD/on-tasks/pull/515, RackHD/ucs-service#24

@iceiilin @anhou @yaolingling @pengz1 @bbcyyb

### CODE REVIEW

File: lib/graphs/ucs-discovery-graph.js [hunk 1]
```diff
@@ -94,6 +94,65 @@ module.exports = {
             }
         },
         {
+            label: 'create-ucs-pollers',
```
Comment (pengz1) [inline lib/graphs/ucs-discovery-graph.js]: Add a user configure options to skip poller, please refer to: https://github.com/RackHD/on-taskgraph/blob/master/lib/graphs/discovery-sku-graph.js

File: spec/lib/graphs/poller-service-graph-spec.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// Copyright 2017, EMC, Inc.
```
Comment (pengz1) [inline spec/lib/graphs/poller-service-graph-spec.js]: // Copyright 2017, EMC, Inc. => // Copyright 2017, Dell EMC, Inc.

**Comment by pengz1** [comment]
Please add a line on the dependency as below:
Jenkins depends on: https://github.com/RackHD/on-tasks/pull/511

Also we should create an independant workflow to create ucs pollers, please refer to:
https://github.com/RackHD/on-taskgraph/blob/master/lib/graphs/create-redfish-chassis-poller-graph.js

**Comment by mcgG** [comment]
Jenkins depends on: https://github.com/RackHD/on-tasks/pull/511

**Comment by mcgG** [comment]
Jenkins depends on: RackHD/on-tasks#511

**Comment by mcgG** [comment]
Jenkins depends on: RackHD/on-tasks#511

**Comment by mcgG** [comment]
test this please

**Comment by mcgG** [comment]
test this please

**Comment by anhou** [comment]
add to whitelist

File: lib/graphs/ucs-discovery-graph.js [hunk 3]
```diff
@@ -94,6 +98,81 @@ module.exports = {
...
+                friendlyName: 'Create Default Pollers',
+                injectableName: 'Task.Inline.Pollers.CreateDefault',
+                implementsTask: 'Task.Base.Pollers.CreateDefault',
+                properties: {},
+                options: {
+                    nodeId: null,
+                    pollers: [
+                        {
+                            "type": "ucs",
+                            "pollInterval": 30000,
```
Comment (pengz1) [inline lib/graphs/ucs-discovery-graph.js]: 30000=>60000

**Comment by pengz1** [comment]
test this please

**Comment by mcgG** [comment]
test this please

**Comment by PengTian0** [comment]
test this please

**Comment by PengTian0** [comment]
test this please

**Comment by pengz1** [comment]
test this please

**Pull request merged**